matrix-sdk-android/src/main/java/im/vector/matrix/android/internal/session/DefaultSession.kt

```diff
@@ -175,6 +183,10 @@ internal class DefaultSession @Inject constructor(
         isOpen = false
         eventBus.unregister(this)
         shieldTrustUpdater.stop()
+        GlobalScope.launch(coroutineDispatchers.main) {
```

Comment:
Avoid GlobalScope

Reply:
How can I execute that on Ui thread then?